lib/rubocop/cop/rspec/cop.rb

```diff
@@ -68,6 +69,10 @@ def rspec_pattern_config
             .fetch('RSpec', DEFAULT_CONFIGURATION)
             .fetch('Patterns')
         end
+
+        def rspec_language_config(*keys)
+          all_cops_config.dig('RSpec', 'Language', *keys)&.map(&:to_sym) || []
```

Comment:
one way to avoid the nil safety and the || is to use `to_a` as nil converts to an empty array
```
all_cops_config.dig('RSpec', 'Language', *keys).to_a.map(&:to_sym)
```

Reply:
Someday I should write a cop to avoid unnecessary nil safety, as I hate seeing &. chains where a conversion will do the job